manim/mobject/geometry/arc.py

```diff
@@ -297,18 +312,18 @@ def construct(self):
 
     def __init__(
         self,
-        radius: float = 1.0,
+        radius: float | None = 1.0,
         start_angle: float = 0,
         angle: float = TAU / 4,
         num_components: int = 9,
-        arc_center: Point3D = ORIGIN,
-        **kwargs,
+        arc_center: InternalPoint3D = ORIGIN,
```

Comment:
This seems to me like a normal `Point3D` should work here?

Reply:
The `arc_center` is later used by the `Mobject.shift` method, which takes a number of `Vector3D`'s as input.
The `Vector3D` is not compatible with the `tuple[float, float, float]` part of `Point3D`.
So either we have to use `InternalPoint3D` here or the type of the `Mobject.shift` method should be changed.